pkgs/development/python-modules/trezor/default.nix

```
@@ -39,12 +46,29 @@ buildPythonPackage rec {
     ecdsa
     libusb1
     mnemonic
+    noiseprotocol
     requests
     shamir-mnemonic
     slip10
     typing-extensions
-  ]
-  ++ lib.optionals stdenv.hostPlatform.isLinux [ trezor-udev-rules ];
+  ];
+
+  optional-dependencies.ble = [ bleak ];
```

I think
```suggestion
  optional-dependencies = {
    ble = lib.optionals stdenv.hostPlatform.isLinux [ bleak ];
```
would be nicer but that's up to you

---

applied in 82ce2223ea4054847637098d479488a7029279b8